renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ]
+}
```

Comment:
Ooooh, so nice to see this :) 🎉

I recommend setting up some groupings to reduce the randomness of PRs e.g.
 * Update plugin com.android.application to v7.4.1
 * Update plugin com.android.library to v7.4.1

are the same thing.

I have done this here:
https://github.com/TWiStErRob/renovate-config/blob/c94936b859a48dbc5910c191bae0b267731b4998/group.json#L102-L135

(Above link shows usage in a shared way, but you can just merge the `packageRules` into your config.)